### PR TITLE
Show call stats in admin trace command

### PR DESCRIPTION
**Pull request by poornas**
companion minio/minio PR https://github.com/minio/minio/pull/7915

output will show call latency, input bytes received in request, output bytes sent in response in addition to trace

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -36,14 +32,15 @@ require (
...
-	golang.org/x/net v0.0.0-20190619014844-b5b0513f8c1b
+	golang.org/x/net v0.0.0-20190628185345-da137c7871d7
 	golang.org/x/text v0.3.2
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
 	gopkg.in/cheggaaa/pb.v1 v1.0.28 // indirect
 	gopkg.in/h2non/filetype.v1 v1.0.5
 )
+
+replace github.com/minio/minio => ../../minio/minio
```
Comment (harshavardhana) [inline go.mod]: Is this dependent on server change ?

Comment (poornas) [reply go.mod]: that's correct - will remove this line after minio/minio PR is merged

**Comment by ebozduman** [comment]
@poornas ,

Please check and fix the following build errors:

```
$ go build
# github.com/minio/mc/cmd
cmd/admin-trace.go:187:15: t.CallStats undefined (type trace.Info has no field or method CallStats)
cmd/admin-trace.go:253:20: t.TraceInfo.Trace.CallStats undefined (type trace.Info has no field or method CallStats)
cmd/admin-trace.go:280:81: trc.CallStats undefined (type trace.Info has no field or method CallStats)
```

**Comment by poornas** [comment]
@ebozduman, travis is failing because a replace directive is used in go.mod for the minio/minio pkg.  We need [this](https://github.com/minio/minio/pull/7915) to be merged first so that Travis passes.

**Comment by ebozduman** [comment]
So, does this mean we'll review this PR after the minio side changes comes in?

**Comment by poornas** [comment]
> So, does this mean we'll review this PR after the minio side changes comes in?
@ebozduman, you can pull in minio/minio#7915 and this PR and review - the replace directive will reference local minio branch.

**Pull request merged**